source/agora/network/Manager.d

```diff
@@ -482,10 +482,10 @@ public class NetworkManager
             if (node_time == 0)
                 continue;  // request failed
 
-            const req_delay = this.clock.utcTime() - req_start;
+            const req_delay = (this.clock.utcTime() - req_start).seconds;
             const dist_delay = req_delay / 2;  // divide evently
-            const offset = (node_time - dist_delay) - req_start;
-            offsets ~= TimeInfo(node.identity.key, node_time, req_delay, offset.seconds);
+            const offset = (node_time - req_start).seconds - dist_delay;
```

Comment:
We should probably rename `getLocalTime`, as it has been returning UTC time for some time.